hazelcast/src/test/java/com/hazelcast/internal/ascii/MemcacheTest.java

```diff
@@ -35,6 +35,7 @@
 import net.spy.memcached.internal.OperationFuture;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
I need to enable it again. I was struggling with memcache due to it poking too much in the internals. This has been addressed; but didn't enable the test again.